paddle/fluid/pybind/global_value_getter_setter.cc

```diff
@@ -33,12 +33,46 @@ DECLARE_bool(use_ngraph);
 DECLARE_bool(use_system_allocator);
 DECLARE_bool(free_idle_chunk);
 DECLARE_bool(free_when_no_cache_hit);
+DECLARE_bool(check_nan_inf);
```

Comment:
It is fine now, but in the future you may group flags together.

Usually, flags/imports are grouped by their functionality first, then alphabetical order

Reply:
Done.